source/about/setup-mattermost-google-calendar-plugin.rst

```diff
@@ -0,0 +1,122 @@
+    :alt: In Google Cloud Console, search for and enable the Google Calendar API.
+
+  .. image:: ../images/enable-google-cloud-console-google-people-api.png
+    :alt: In Google Cloud Console, search for and enable the Google People API.
+
+3. Choose how to configure and register the application by selecting the user type as **Internal** or **External**, then select **Create**.
+
+  .. image:: ../images/google-cloud-console-OAuth-consent-screen.png
+    :alt: On the OAuth consent screen, configure and register the application as internal or external.
+
+4. When prompted, set the following app information:
+
+  - **App name**: `Google Calendar Mattermost Plugin`
+  - **User support email** 
+  - **Developer contact information**
+  - Populate the remaining fields as needed.
+
+  .. image:: ../images/google-cloud-console-app-information.png
+    :alt: Configure the consent screen app name and user support email.
+
+  .. image:: ../images/google-cloud-console-developer-contact-information.png
+    :alt: Configure the consent screen developer contact information.
+
+5. Under Credentials, create new OAuth 2.0 credentials by selecting **Create Credentials > OAuth client ID**.
+
+  - Under **Application type**, select **Web Application**.
+  - Under **Authorized redirect URIs** add ``https://(MM_SITE_URL)/plugins/com.mattermost.gcal/oauth2/complete`` and replace ``MM_SITE_URL`` with your Mattermost Server Site URL.
+
+  .. image:: ../images/google-cloud-console-web-application.png
+    :alt: Under Credentials, select Web application.
+
+  .. image:: ../images/google-cloud-console-authorized-redirect-uris.png
+    :alt: Configure the authorized redirect URI for Mattermost.
+
+6. Once the OAuth client is created, make a copy of the **Client ID** and **Client Secret** values for the next and final setup step below.
+
+  .. image:: ../images/google-cloud-console-OAuth-client-created.png
+    :alt: Copy the Client ID and Client secret values. You'll need these values in the next step.
+
+Configure the Mattermost Google Calendar plugin
+-----------------------------------------------
+
+In Mattermost, configure the plugin by going to **System Console > Plugin Management > Google Calendar**, enter the following values, and select **Save**.
+
+- **Enable Plugin**: Select **true** to enable the plugin for your Mattermost instance.
+- **Admin User IDs**: Specify the user IDs who are authorized to manage the plugin in addition to Mattermost system admins. Separate multiple user IDs with commas. Go to **System Console > User Management > Users** to obtain a user's ID.
+- **Encryption key**: Generate an encryption key used to store data in the database. Regenerating this value forces users to re-link their Google Calendars in Mattermost.
+- **Google Application Client ID**: Paste the Client ID value from the Google Cloud Console.
+- **Google Client Secret**: Paste the Client Secret value from the Google Cloud Console.
+- **Google Domain verify key**: TBD
```

Comment:
We are removing this setting.

https://mattermost.atlassian.net/browse/MM-54152
https://github.com/mattermost/mattermost-plugin-mscalendar/pull/299

```suggestion
```

Reply:
Thanks! Removed!